<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogArcOctantEdit</class>
 <widget class="QDialog" name="DialogArcOctantEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Arc Octant</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSpinBox" name="spBox_StartOctant">
          <property name="maximum">
           <number>7</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="spBox_NumberOctant">
          <property name="minimum">
           <number>-7</number>
          </property>
          <property name="maximum">
           <number>7</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Number</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QSpinBox" name="spBox_Radius">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>256</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Radius</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogArcOctantEdit</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogArcOctantEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>158</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spBox_StartOctant</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogArcOctantEdit</receiver>
   <slot>acrOctantStartChadged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spBox_NumberOctant</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogArcOctantEdit</receiver>
   <slot>acrOctantNumberChadged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spBox_Radius</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogArcOctantEdit</receiver>
   <slot>acrOctantRadiusChadged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>acrOctantStartChadged(int)</slot>
  <slot>acrOctantNumberChadged(int)</slot>
  <slot>acrOctantRadiusChadged(int)</slot>
 </slots>
</ui>
